objects.ron

```diff
@@ -130,7 +142,9 @@
         ]),
     ],
     "hammerman": [
-        Blocker(()),
+        Blocker((
+            weight: Normal,
```

Comment:
Hmm. If you put `#[serde(default)]` annotation on the `weight` field declaration it should be possible to omit all these `weight: Normal` repetitions.